static/app/views/alerts/wizard/options.tsx

```diff
@@ -273,6 +274,13 @@ const INDEXED_PERFORMANCE_ALERTS_OMITTED_TAGS = [
   ...Object.values(ReplayClickFieldKey),
 ];
 
+const ERROR_SUPPORTED_TAGS = [
```

Comment:
@rachrwang could you please confirm that all the tags you were looking for are now listed with my changes?

Reply:
Tags look good to me, thanks Priscila!